.happy/terraform/modules/ecs-stack/main.tf

```diff
@@ -33,17 +32,16 @@ locals {
   aspen_data_bucket     = local.secret["s3_buckets"]["aspen_data"]["name"]
 
   # Web images
-  frontend_image_repo   = local.secret["ecrs"]["frontend"]["url"]
-  backend_image_repo    = local.secret["ecrs"]["backend"]["url"]
+  frontend_image   = join(":", [local.secret["ecrs"]["frontend"]["url"], lookup(var.image_tags, "frontend", var.image_tag)])
+  backend_image    = join(":", [local.secret["ecrs"]["backend"]["url"], lookup(var.image_tags, "backend", var.image_tag)])
 
   # Workflow images
-  pangolin_image_repo   = local.secret["ecrs"]["pangolin"]["url"]
-  nextstrain_image_repo = local.secret["ecrs"]["nextstrain"]["url"]
-  gisaid_image_repo     = local.secret["ecrs"]["gisaid"]["url"]
-  covidhub_import_image_repo  = local.secret["ecrs"]["covidhub-import"]["url"]
+  pangolin_image   = join(":", [local.secret["ecrs"]["pangolin"]["url"], lookup(var.image_tags, "pangolin", var.image_tag)])
+  nextstrain_image = join(":", [local.secret["ecrs"]["nextstrain"]["url"], lookup(var.image_tags, "nextstrain", var.image_tag)])
+  gisaid_image     = join(":", [local.secret["ecrs"]["gisaid"]["url"], lookup(var.image_tags, "gisaid", var.image_tag)])
```

Comment:
Simplifying the way we refer to these images in the rest of our tf code.

We look to see whether the image name is included in our list of tag overrides (`var.image_tags`) and if it's not overridden, we use the default tag.